src/acceptance/api/api_suite_test.go

```diff
@@ -88,7 +88,7 @@ var _ = BeforeSuite(func() {
 	WaitForNInstancesRunning(appGUID, initialInstanceCount, cfg.DefaultTimeoutDuration())
 
 	if cfg.IsServiceOfferingEnabled() {
-		serviceExists := cf.Cf("marketplace", "-s", cfg.ServiceName).Wait(cfg.DefaultTimeoutDuration())
+		serviceExists := cf.Cf("marketplace", "-e", cfg.ServiceName).Wait(cfg.DefaultTimeoutDuration())
```

Comment:
What if someone still use the v6 cli?  I tried to run the acceptance test with CLIv6, this test case will failed.

Reply:
Resolved in ed3ad03